src/ScHelpers.ts

```diff
@@ -0,0 +1,158 @@
+import { ScAddr } from "./ScAddr";
+import { ScClient } from "./ScClient";
+import { ScConstruction } from "./ScConstruction";
+import { ScEventType } from "./ScEvent";
+import { ScEventParams } from "./ScEventParams";
+import { ScLinkContent, ScLinkContentType } from "./ScLinkContent";
+import { ScTemplate } from "./ScTemplate";
+import { ScType } from "./ScType";
+import { snakeToCamelCase } from "./utils";
+
+export class ScHelpers {
+  private _client: ScClient;
+
+  constructor(client: ScClient) {
+    this._client = client;
+  }
+
+  public async getMainIdLinkAddr(addr: ScAddr, lang: string) {
```

Comment:
We usually name such function with `idtf` instead of `id`. Please update here and later